index.js

```diff
@@ -1,8 +1,9 @@
 const Discord = require('discord.js');
 const client = new Discord.Client();
 
+process.env.DISCORD_TOKEN = "insertDiscordTokenHere"
```

Comment:
Hmm, is there a reason we can't use environment variables in Windows? (I presume you're doing this in Windows from the changes to the README.) I can probably help you out with that if it's just lack of knowledge.

That said, I dislike this change to the code because it would be really easy to mistakenly forget your token is in here when pushing a change to git. There are bots out there that constantly roam the internet looking for discord tokens and other types of passwords, and they will do nasty things within minutes of you uploading your code. Safer to keep it completely separate- if not in an environment variable, than in another file with a corresponding entry in a .gitignore file. Also, for what it's worth, adding the token directly in the code makes debugging far harder for me...

I'd love to hear your thoughts on this. Also, thanks for contributing! I'm super excited that this project got its first PR!

Reply:
Another bot I've toyed with used a json config file that was in the .gitignore for the token. Would that suit you better? I appreciate a good ENV variable though. Just not running on MacOS anymore, although my VM hosting ill be Ubuntu.....

And excited to be the first! I want to use this run a Discord server with Play by Post.

Reply:
Also, are you on discord? I thought I found your repo via Discord. I'm at mirkoRainer#3102

Reply:
One more thing.
Would you be cool with a testing framework for this?
Mocha or Chai?

Reply:
OMG yes. There's actually an open issue for figuring out how to add a automated testing framework for this. Due to its nature though, I haven't found a suitable method for doing it. I've already accidentally caused regression bugs in the bot with some of my changes though, so I know there's an immediate need for _something_. I haven't heard of those two testing frameworks though. Do you have experience with either?

A JSON file with a .gitignore would be totally fine, if not welcome. I could see that being easier for new users.